.github/workflows/ci.yml

```
@@ -484,6 +484,27 @@ jobs:
             pandas \
             pytest
 
+      - name: Install wait-for-it
+        run: |
+          sudo apt-get update -y && sudo apt-get install -y wait-for-it
+
+      - name: Download the Docker containers
+        run: |
+          docker-compose pull
+
+      - name: Start the Docker containers
+         run: |
```

a GitHub no le gustó este espacio de más: https://github.com/omegaup/omegaup/actions/runs/1799523945


```suggestion
        run: |
```